tests/observability/metrics/conftest.py

```diff
@@ -436,13 +438,14 @@ def windows_vm_info_to_compare(windows_vm_for_test):
 
 
 @pytest.fixture(scope="module")
-def windows_vm_for_test(namespace, unprivileged_client):
+def windows_vm_for_test(namespace, unprivileged_client, admin_client):
     with create_windows11_wsl2_vm(
         dv_name="dv-for-windows",
         namespace=namespace.name,
         client=unprivileged_client,
         vm_name="win-vm-for-test",
         storage_class=py_config["default_storage_class"],
+        admin_client=admin_client,
```

Comment:
you don't need admin client here... and btw, you also pass unpriv client in line #445